packages/HtmlAgilityPack.1.4.9/HtmlAgilityPack.1.4.9.nuspec

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
-  <metadata>

```

Comment:
I had some problems with HtmlAgilityPack after I forked and cloned the repo, so I used NuGet to uninstall and reinstall HtmlAgilityPack. Since doing this Robotstove, Baconcatbug, and I have had no issues, so I guess it's not important.